dotnet/src/Experimental/Agents/Extensions/AssistantsKernelFunctionExtensions.cs

```diff
@@ -86,4 +85,17 @@ private static string ConvertType(Type? type)
 
         return type.Name;
     }
+
+    private static bool IsNumber(this Type type) =>
```

Comment:
Is this being used anywhere?

Reply:
13 lines above

Reply:
Oh, so is these replacing some helper from JsonSchema.NET?

Reply:
Yup.

Reply:
If this is being used to generate schemas, it should at some point be delegated to STJ. It's leaving out the new numeric types and custom converters can change what is regarded as a number and what isn't.

Reply:
> it should at some point be delegated to STJ

What's the recommended way to do that?

Reply:
There are no public APIs available right now for this, but if you wanted you could call the schema mapper and look up the `type` property of the resultant document.